.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "dependabot"
```

Comment:
There's an existing `dependencies` label - maybe worth renaming it to `dependabot` prior to merging this, for continuity of the already labelled PRs? :-)
https://github.com/heroku/heroku-buildpack-jvm-common/issues?q=label%3Adependencies